### PR TITLE
Add one note when regenerating a new Gemfile.lock

### PR DESCRIPTION
**Pull request by AbelHu**

### CODE REVIEW

**Comment by cppforlife** [comment]
Instead of adding a readme section let's just place these two constraints (specinfra(1.15.0) and serverspec(0.15.4)) into the Gemfile so that bundler will only use those versions.

**Comment by AbelHu** [comment]
Update PR: Specify two constraints (specinfra(1.15.0) and serverspec(0.15.4)) in Gemfile.

**Comment by AbelHu** [comment]
Ping.

File: Gemfile [hunk 1]
```diff
@@ -40,7 +40,7 @@ end
 group :development, :test do
   gem 'bosh-dev', path: 'bosh-dev'
   gem 'bosh-stemcell', path: 'bosh-stemcell'
-  gem 'serverspec'
+  gem 'serverspec', :git => 'git://github.com/serverspec/serverspec.git', :tag => 'v0.15.4'
```
Comment (cppforlife) [inline Gemfile]: Why not do?

```
gem 'serverspec', '0.15.4'
```

Same below.

**Comment by AbelHu** [comment]
Updated it as you said.

**Comment by AbelHu** [comment]
Close and recreate a PR.

**Comment by cppforlife** [comment]
Merged to develop.

**Pull request closed without merging**